keyboards/idb_60/idb_60.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "quantum.h"
+
+inline void _idb_60_caps_led_on(void) {
+    writePinLow(C5);
+}
+
+inline void _idb_60_esc_led_on(void) {
+    writePinLow(C4);
+}
+
+inline void _idb_60_caps_led_off(void) {
+    writePinLow(C5);
+}
+
+inline void _idb_60_esc_led_off(void) {
+    writePinLow(C4);
+}
+
+inline void keyboarb_pre_init_kb(void) {
```

Comment:
spell check

```suggestion
inline void keyboard_pre_init_kb(void) {
```

Reply:
Whoops, sorry.

Reply:
Also, this shouldn't be "inline"

Reply:
Oh, that slipped past me. Fixed.

Reply:
Yeah, slipped past me too, until the second check.